views/js/controller/Delivery/monitoring.js

```diff
@@ -560,6 +528,50 @@ define([
                 }
             });
 
+            // column: pause
+            model.push({
+                id: 'pause',
+                label: __('Pause'),
+                transform: function(value, row) {
+                    var status;
+                    var inactive = true;
+
+                    if (row.state && row.state.status) {
+                        status = _status.getStatusByCode(row.state.status);
+                        inactive = !status || status.can.pause !== true;
+                    }
+
+                    return actionBtnTpl({
+                        icon: 'pause',
+                        type: 'pause',
+                        title: __('Pause session'),
+                        disabled: inactive ? 'disabled' : ''
+                    });
+                }
+            });
+
+            // column: pause
+            model.push({
+                id: 'authorize',
+                label: __('Authorize'),
+                transform: function(value, row) {
+                    var status;
+                    var inactive = true;
+
+                    if (row.state && row.state.status){
+                        status = _status.getStatusByCode(row.state.status);
+                        inactive = !status || status.can.authorize !== true;
+                    }
+
+                    return actionBtnTpl({
```

Comment:
I was actually expected to have this action button as part of a configurable option for the datatable like the last column action... so modifying the datatable component to allow inserting action not only in the end but also the location we want. This way we don't have to redefine views/js/templates/delivery/actionBtn.tpl each time and add it from the  transform() callback.
I am not sure it is easily feasible but since you are ahead of schedule, what about exploring this option ?